roles/convert2rhel/tasks/products_and_repos.yml

```diff
@@ -52,6 +52,24 @@
             state: present
   when: foreman_content_rhel_enable_rhel8
 
+- name: "Create 'UBI' product and repositories"
```

Comment:
what is this repo and why is this needed? (a doc link to make my brain happy is sufficient)

Reply:
OL does not have subscription-manager available in its repositories, but we need that for the registration, the best source for that is the [UBI](https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image) repository. Convert2rhel installs the subscription-manager from there when it's not used with Satellite (and expects the Internet connectivity)

Reply:
I think we should call the product/repos differently. While they come from UBI, this is not what we use them for and this will lead to confusion.

How about "Oracle Linux subscription-manager dependencies" or something? Or "Oracle Linux convert2rhel dependencies"?